questdb-rs/src/ingress/mod.rs

```diff
@@ -909,6 +909,57 @@ impl Buffer {
         self.output.push(b'i');
         Ok(self)
     }
+    /// Record an integer value for the given column.
+    ///
+    /// ```
+    /// # use questdb::Result;
+    /// # use questdb::ingress::Buffer;
+    /// # fn main() -> Result<()> {
+    /// # let mut buffer = Buffer::new();
+    /// # buffer.table("x")?;
+    /// buffer.column_u64("col_name", 42)?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    ///
+    /// or
+    ///
+    /// ```
+    /// # use questdb::Result;
+    /// # use questdb::ingress::Buffer;
+    /// use questdb::ingress::ColumnName;
+    ///
+    /// # fn main() -> Result<()> {
+    /// # let mut buffer = Buffer::new();
+    /// # buffer.table("x")?;
+    /// let col_name = ColumnName::new("col_name")?;
+    /// buffer.column_u64(col_name, 42);
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn column_u64<'a, N>(&mut self, name: N, value: u64) -> Result<&mut Self>
+    where
+        N: TryInto<ColumnName<'a>>,
+        Error: From<N::Error>,
+    {
+        self.write_column_key(name)?;
+        let mut buf = itoa::Buffer::new();
+        let printed = buf.format(value);
+        self.output.extend_from_slice(printed.as_bytes());
+        self.output.push(b'i');
+        Ok(self)
+    }
```

Comment:
It would be misleading for the API to accept a datatype that the server does not.
If you need to serialize a `u64`, best do a checked cast and then call `column_i64`.